### PR TITLE
New functions Graphic and Table

### PR DESCRIPTION
**Pull request by CristianoAugustoChico**
New functions Graphic and Table

### CODE REVIEW

File: tests/functional/basic/Report.lua [hunk 1]
```diff
@@ -29,23 +29,33 @@ return {
...
+		unitTest:assertEquals(getn(report.graphic), 0)
+		unitTest:assertEquals(getn(report.mult), 0)
 
 		local image = packageInfo("publish").path.."images/urbis_2010_real.PNG"
 		report = Report{title = "My Report"}
 
 		report:addHeading("My Heading")
 		report:addImage(image)
 		report:addText("Some text")
+		report:addMatrix("My table")
```
Comment (pedro-andrade-inpe) [inline tests/functional/basic/Report.lua:42]: addMatrix does not exist. Please remove this and the other tests that call this function.

Comment (CristianoAugustoChico) [reply tests/functional/basic/Report.lua:42]: Okay

File: tests/functional/basic/Report.lua [hunk 2]
```diff
@@ -155,6 +154,21 @@ return {
 	end,
 	addMatrix = function(unitTest)
```
Comment (pedro-andrade-inpe) [inline tests/functional/basic/Report.lua:155]: This tests must me removed, as there is no funcion with this name.

Comment (CristianoAugustoChico) [reply tests/functional/basic/Report.lua]: I removed all test from the new functions.

**Pull request merged**